docs/docs/en/guide/installation/pseudo-cluster.md

```diff
@@ -187,7 +187,7 @@ bash ./bin/dolphinscheduler-daemon.sh stop alert-server
 
 > **_Note1:_**: Each server have `dolphinscheduler_env.sh` file in path `<server-name>/conf/dolphinscheduler_env.sh` which
 > for micro-services need. It means that you could start all servers by command `<server-name>/bin/start.sh` with different
-> environment variable from `bin/env/dolphinscheduler_env.sh`. But it will use file `bin/env/dolphinscheduler_env.sh` overwrite
+> environment variable from `<server-name>/conf/dolphinscheduler_env.sh`. But it will use file `bin/env/dolphinscheduler_env.sh` overwrite
```

Comment:
I think we should use `<service>`to ensure consistency with `https://dolphinscheduler.apache.org/zh-cn/docs/latest/user_doc/guide/installation/kubernetes.html`